Proposal Review #1 (#53)
lhossackucalgary/ENGO-500-2018: proposal-template.md

> Provide a short narrative description of the project.  Identify/describe any technical and business problem that would be solved, the key benefits and how the project aligns with/supports the strategic direction/objectives of the proponent organization or members, its collaborators, and clients/users.

## Proponent Profile

> RouteScouts is an organization made up of four members: Lucas Hossack, Deanna Wing Yan Ip, Shuet-Ching Christina Lo, and Joan Ching Kwan Tam. All members are currently in their last year of a Geomatics Engineering degree, as well as one member (Lucas) finishing a double major in Computer Science and two members (Christina and Joan) finishing a minor in Computer Science. We have a collective interest in programming, Internet of Things (IoT), routing algorithms, innovations in tech regarding the use and manipulation of geospatial data, as well as AR. Using the skills and knowledge we have gained from our courses in Geomatics and Computer Science, we are aiming to improve the efficiency of mobile maintenance and support teams. We hope to also incorporate our interests by integrating a routing system, visualizing geospatial and sensor data on maps, and AR into our project.

Reviewer: ```suggestion
RouteScouts is an organization made up of four members: Lucas Hossack, Deanna Wing Yan Ip, Shuet-Ching Christina Lo, and Joan Ching Kwan Tam. All members are currently in their last year of a Geomatics Engineering degree, as well as one member (Lucas) finishing a combined degree in Computer Science and two members (Christina and Joan) finishing a minor in Computer Science. We have a collective interest in programming, Internet of Things (IoT), routing algorithms, innovations in tech regarding the use and manipulation of geospatial data, as well as AR. Using the skills and knowledge we have gained from our courses in Geomatics and Computer Science, we are aiming to improve the efficiency of mobile maintenance and support teams. We hope to also incorporate our interests by integrating a routing system, visualizing geospatial and sensor data on maps, and AR into our project.
```

> Specify any major policy, standards, data, technical, partnership and/or client engagement issues that need to be addressed in order for the project to be successful. 
The issues that will affect the success and completion of our project have been identified as the following:

> 1. We have very limited knowledge regarding machinry and their maintenance. 

Reviewer: ```suggestion
1. We have very limited knowledge regarding machinery and their maintenance. 
```

Communication on the progress of the project between our organization, collaborating organizations, and clients/users will be done during major milestones throughout the next 6-7 months. These milestones include the mid-semester progress report presentations/reports, the final presentation/reports, as well as the Capstone Fair. In the progress reports we will be showcasing the work that we have completed, where we are in the context of our project implementation plan, and future work to be done. The presentations for these progress reports will be presented in front of a spokesperson with our collaborating organization SensorUp and project advisor, Steve Liang. The final report will contain in-depth details on our completed project and we will be presenting our completed project both in the final presentation and at the Capstone Fair. In the final presentations, we will again be presenting in front of Dr.Liang. For the Capstone Fair, we will be showing the final project results to our clients and users, as well as prospective clients for the future.

> Within our organization and collaborating organizations, we are using messaging applications in order to share and disseminate results. For example we have kept in constant communication with Dr.Liang using Slack. We share progress in our research and project on a regular basis and organize meetings with Dr. Liang using Slack. Internally as RouteScouts, we use Slack to track individual progress as well as plan meetings with the rest of the organization. For example, in order to approve a pull request on GitHub, we message each other that a change has been made, and to ask other members to review the request. Other members will then be aware of this pull request and will approve if the changes are acceptable and have been tested. This method of internal communication will continue to be used throughout this project at a greater frequency as we move into writing code and ensuring there are no project conflicts. 

Reviewer: ```suggestion
Within our organization and collaborating organizations, we are using messaging applications in order to share and disseminate results. For example we have kept in constant communication with Dr. Liang using Slack. We share progress in our research and project on a regular basis and organize meetings with Dr. Liang using Slack. Internally as RouteScouts, we use Slack to track individual progress as well as plan meetings with the rest of the organization. For example, in order to approve a pull request on GitHub, we message each other that a change has been made, and to ask other members to review the request. Other members will then be aware of this pull request and will approve if the changes are acceptable and have been tested. This method of internal communication will continue to be used throughout this project at a greater frequency as we move into writing code and ensuring there are no project conflicts. 
```
